### ISSUE TITLE
Ability to parse multiple maintenance windows from Zayo maintenance notification 

### ISSUE THREAD

**Issue by aliex-13** [networktocode/circuit-maintenance-parser#210]
### Environment

- circuit_maintenance_parser version: v2.2.2

Return multiple maintenance windows in the event where a vendor sends us multiple activity dates. This feature should allow multiple maintenance windows to be returned as separate fields rather than one big window. Currently, multiple windows from vendor are being clumped into one large window (earliest start time + latest end time found).

### Proposed Functionality

The proposed functionality of this would be allowing maintenance windows to only be active for the window that the vendor specifies, not all the time in between the earliest start time and latest end time. This increases availability of network resources in the event where operators move traffic away from network links during maintenance windows.

### Use Case

An example email (sanitized) that we would receive:

```
                # Some Zayo notifications may include multiple activity dates.
                # For example, given:
                #
                # 1st Activity Date
                # 01-Nov-2021 00:01 to 01-Nov-2021 05:00 ( Mountain )
                # 01-Nov-2021 06:01 to 01-Nov-2021 11:00 ( GMT )
                #
                # 2nd Activity Date
                # 02-Nov-2021 00:01 to 02-Nov-2021 05:00 ( Mountain )
                # 02-Nov-2021 06:01 to 02-Nov-2021 11:00 ( GMT )
                #
                # 3rd Activity Date
                # 03-Nov-2021 00:01 to 03-Nov-2021 05:00 ( Mountain )
                # 03-Nov-2021 06:01 to 03-Nov-2021 11:00 ( GMT )
```

By testing the email file against circuit-maintenance-parser, we should be getting multiple maintenance objects rather than just one with the earliest start time and latest end time:

```
#circuit-maintenance-parser --data-file ~/zayo_test_email.eml --data-type html --provider-type zayo

Circuit Maintenance Notification #0
{
  "account": "Test",
  "circuits": [
    {
      "circuit_id": "ZAYO/TEST/CIRCUIT",
      "impact": "OUTAGE"
    }
  ],
  "end": 1635764400,
  "maintenance_id": "TTN-0001234567",
  "organizer": "mr@zayo.com",
  "provider": "zayo",
  "sequence": 1,
  "stamp": 1635573660,
  "start": 1635746460,
  "status": "CONFIRMED",
  "summary": "Zayo Third-Party Provider will implement maintenance to perform temporary fiber relocation in order to proactively avoid outages.",
  "uid": "0"
}

Circuit Maintenance Notification #1
{
  "account": "Test",
  "circuits": [
    {
      "circuit_id": "ZAYO/TEST/CIRCUIT",
      "impact": "OUTAGE"
    }
  ],
  "end": 1635850800,
  "maintenance_id": "TTN-0001234567",
  "organizer": "mr@zayo.com",
  "provider": "zayo",
  "sequence": 1,
  "stamp": 1635573660,
  "start": 1635832860,
  "status": "CONFIRMED",
  "summary": "Zayo Third-Party Provider will implement maintenance to perform temporary fiber relocation in order to proactively avoid outages.",
  "uid": "1"
}

Circuit Maintenance Notification #2
{
  "account": "Test",
  "circuits": [
    {
      "circuit_id": "ZAYO/TEST/CIRCUIT",
      "impact": "OUTAGE"
    }
  ],
  "end": 1635937200,
  "maintenance_id": "TTN-0001234567",
  "organizer": "mr@zayo.com",
  "provider": "zayo",
  "sequence": 1,
  "stamp": 1635573660,
  "start": 1635919260,
  "status": "CONFIRMED",
  "summary": "Zayo Third-Party Provider will implement maintenance to perform temporary fiber relocation in order to proactively avoid outages.",
  "uid": "2"
}
```

**Comment by chadell** [comment]
@aliex-13 could you provide an anonymized email example with this format?

**Comment by csessh** [comment]
@chadell
Feel free to assign this issue to either of us

**Comment by chadell** [comment]
Fixed in #214 by @csessh
